packages/developer-console-spa/README.md

```diff
@@ -0,0 +1,38 @@
+# One Platform Developer Console
+
+The One Platform Developer Console is the onboarding hub for any developer looking to host or integrate their apps with One Platform. It provides quick and easy ways to setup and deploy your apps on the Platform, and simple configurations to enable and integrate One Platform Microservices with your apps.
+
+## Setup
+
+To setup the config, just copy the `.env.example` file, rename it as `.env` and provide a URL for the API Gateway using the `REACT_APP_API_GATEWAY` variable.
```

Comment:
Mention about the ```PUBLIC_URL``` configuration

Reply:
We don't need to define the `PUBLIC_URL`. React uses the value of the homepage property from `package.json`.